### ISSUE TITLE
Validation errors not shown

### ISSUE THREAD

**Issue by ivelin** [RestComm/visual-designer#5]
I encountered a situation where RVD shows Validation Error message, but does not provide detail as to the specific origin and details of the error. It just displays a yellow exclamation mark on the Record component, which does not tell me much.

<img width="644" alt="screen shot 2017-02-12 at 1 46 23 pm" src="https://cloud.githubusercontent.com/assets/2234901/22866333/e3b9ff6e-f129-11e6-9f14-8a1f5a36b81d.png">

@otsakir @ipsilantide @deruelle

**Comment by FerUy** [comment]
In this case, it's relatively easy to find the problems, as they are highlighted in red within the module (it could tell more anyway, I agree). IMHO what it would be more helpful is to indicate the modules with errors in the message on the top, something like "Project saved with validation errors in record verb of module Welcome", especially when the project has a lot of modules and verbs with errors. Maybe this could be another issue?

**Comment by otsakir** [comment]
@ivelin, i totally agree. 

I've started addressing this issue starting with the new RVD _control_ element. A popup is displayed when clicking on the exclamation mark explaining exactly what is the problem. The same will be applied to the rest of the elements. Also, further indicating errors on the module tab will help have an quick overview of the errors.

**Comment by otsakir** [comment]
This also depends on #14
